### PR TITLE
Revise toolbar status styles

### PR DESCRIPTION
**Pull request by terotic**
Maybe an overkill with icons and variating alert styles, but let's try this.

### CODE REVIEW

File: src/components/admin/HearingToolbar.js [hunk 1]
```diff
@@ -38,7 +36,7 @@ class HearingToolbar extends React.Component {
     ];
     const open = isOpen(hearing);
     if (open && hearing.published) {
-      statusLabel = <FormattedMessage id="published"/>;
+      statusLabel = (<Alert bsStyle="success"><Icon name="eye"/> <FormattedMessage id="published"/></Alert>);
```
Comment (Rikuoja) [inline src/components/admin/HearingToolbar.js]: Don't know if the backgrounds are overkill, but I think the green background is too dark on my display. The black text "Julkaistu" doesn't stand out as it should. The lighter backgrounds work alright.

Comment (terotic) [inline src/components/admin/HearingToolbar.js]: Ok. Should also file an issue about this in hel-bootstrap-3. Must use white text with success, like we use in danger.

Comment (Rikuoja) [inline src/components/admin/HearingToolbar.js]: Ah. Should test the white text before filing an issue though.

Comment (Rikuoja) [inline src/components/admin/HearingToolbar.js]: Oh, and did you style the (non-react) notify.js notifications too? The ones that pop up when you post things in the UI?

Comment (terotic) [inline src/components/admin/HearingToolbar.js]: You mean the Alertify popups? I can have a look, I guess they are using the plugin's default style at the moment.

Comment (Rikuoja) [inline src/components/admin/HearingToolbar.js]: Ah it was Alertify, those exactly.

**Pull request merged**